docs/cudf/source/user_guide/api_docs/pylibcudf/strings/index.rst

```diff
@@ -12,6 +12,7 @@ strings
     find_multiple
     findall
     padding
+    json
```

Comment:
The `get_json_object` and related APIs are not part of the `cudf::strings` namespace in libcudf.
https://github.com/rapidsai/cudf/blob/branch-24.12/cpp/include/cudf/json/json.hpp
Should pylibcudf match libcudf more closely than cudf?

Reply:
Yes pylibcudf should match libcudf more closely than cudf python. So, I think we should move `json.pxd` and `json.pyx` out of strings.

But a broader point: I think pylibcudf is first and foremost ["python bindings for libcudf"](https://docs.rapids.ai/api/cudf/stable/developer_guide/pylibcudf/#pylibcudf), so we should match libcudf as closely as possible. But pylibcudf is a python library so I think it makes sense over time to make it more "pythonic" (eg. easier to use with other python libraries).
cc. @vyasr

Reply:
Thanks for the catch. I moved this out of the `string` namespace.

Reply:
> Or were you just responding to David's question with a statement that while pylibcudf APIs should closely match libcudf APIs, they may not match exactly if there is a more Pythonic choice for the pylibcudf API that still faithfully reflects the semantics of the C++ functions?

That's right. I realized I didn't add "Is that right?" before I @'d you because I wanted to get your thoughts.